Documentation/markdown/UnitTests.md

```diff
@@ -0,0 +1,24 @@
+# Unit Tests
+
+OIP already provides a set of unit tests to check continuously that even after changes 
+to the code, everything is still working smoothly.
+Primarily official buildingSMART IFC example files are used to check OIP for correctness
+since they come with screenshots and additional information on how a correct visualization needs to 
+look.
+In this way, specific IfcEntities can be inspected individually for correct implementation.
+
+Currently, the OIP unit tests are not automated and need to be run manually.
+In order to run them, make sure you selected the unit test module in the CMake configuration options
+(for more information on the CMake configuration, take a look at the CMakeOptions).
```

Comment:
link to cmake options?

do we really have such option?

Reply:
I already had the link in there but accidentally removed it.

The BUILD_TESTS option should do that.